ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/PartyManagementServiceIT.scala

```diff
@@ -449,89 +446,4 @@ final class PartyManagementServiceIT extends PartyManagementITBase {
     }
   })
 
-  test(
-    "UpdateAnnotationsOfNonLocalParty",
-    "Update annotations of a non-local party",
```

Comment:
This test will fail when running 2.5.0 tool against 2.5.1 canton, we'd better get ready to silence it

Reply:
Indeed, such a removal makes our protocol continuity test fail :) We will add an exception.

Reply:
Did it trigger for you already? I would like to at least know which places these are :)

Reply:
Yes, in the corresponding Daml update PR (https://github.com/DACH-NY/canton/pull/11285), and more precisely in the `protocol_continuity_test_latest` (https://app.circleci.com/pipelines/github/DACH-NY/canton/43595/workflows/c30ccb72-bfe3-45bf-ac66-0832944c886a/jobs/983226).

There, we try to run conformance test using:

- Domain on the latest release (in this case: 2.5.0)
- Participant on current commit

Invoking the `ledger-api-test-tool` with version 2.5.0 to list the tests yields a list that contains UpdateAnnotationsOfNonLocalParty which is not available anymore.